content/docs/learning-guide/tutorials/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,219 @@
+
+The `3DOctree` entity's `Octree component` is an configured for accuracy at the cost of some performance, using a small cell size to represent the complex geometry of the level.
+
+![Octree debug draw](/images/learning-guide/tutorials/ai/kythera-city-octree.png)
+
+*   To generate the octree use the **Generate Octree** icon on the Kythera AI Toolbar,
+    then make sure to save the octree using the **Save Octree** button<br>
+    ![Regenerate Octree toolbar icon](/images/learning-guide/tutorials/ai/kythera-city-octree-regenerate.png)
+*   To show debug draw for the Octree, enter the following commands in the O3DE console:
+    *   `kyt_drawmaster 1`
+    *   `kyt_drawnavoctree 2`
+
+
+![Enabling octree debug draw in the O3DE console](/images/learning-guide/tutorials/ai/kythera-city-octree-console.png)
+
+## Kythera AI Inspector tool (behavior trees and debug draw)
+
+The [Kythera AI Inspector](/docs/user-guide/gems/reference/kythera-ai/introduction-to-the-inspector) is a web-based tool that allows you to author behavior trees (BTs) and access advanced debug draw features. To launch the Kythera AI Inspector, click on the **Launch Inspector** icon in the Kythera Toolbar.
+
+By default, the Inspector will show the blackboard view, where you can debug existing blackboard values, and enable or disable the debug draw options.
+
+![The blackboard view in the Kythera AI Inspector](/images/learning-guide/tutorials/ai/kythera-city-inspector-bbview.png)
+
+To enable debug draw, expand Console Variables or drag it to the side to create a new tab.
+Set console variable `Draw Master` to 1 and collapse console variables.
+
+Expand `debug options`, here you can disable/enable various specific debug draw features.
+
+To view the behavior trees, click on BT Editor at the top of the page and select a behavior tree name to edit.
+Make sure the Open3D game or editor isn't playing or simulating while editing the behavior trees.
+
+![](/images/learning-guide/tutorials/ai/kythera-city-inspector-bteditor.png)
+
+Alternatively, you can view the behavior in live mode, go to:
+
+1. In the top menu bar choose the `Live` tab, then the `Behavior trees` tab below it
+2. Simulate or play the level in the Open3D editor. The Inspector will show `Kythera is active`
+3. Find an entity in the dropdown menu to see which behavior tree is in use.
+4. Click the `Live` button to the right of the dropdown menu to view the behavior tree in real time
+
+## 2D pathfinding
+
+![A 2D Kythera AI agent](/images/learning-guide/tutorials/ai/kythera-city-robot.png)
+
+2D Navigation is demonstrated by the `Robot.prefab`. The robots use the behavior `2DRandomWalk`, which chooses a random point in a specified range, then finds and follows a path to that point.
+
+When duplicating or moving entities, be sure to select the top (shaded tab) of the prefab in the Entity Outliner, to prevent detached objects in the level.
```

Comment:
```suggestion
When duplicating or moving entities, to prevent detached objects in the level, be sure to select the top (shaded tab) of the prefab in the Entity Outliner.
```
Re-ordered to emphasize the purpose of the action first.